onnxruntime/python/onnxruntime_pybind_state.cc

```diff
@@ -763,7 +763,7 @@ bool CheckIfTensor(const std::vector<const NodeArg*>& def_list,
     defined(USE_CUDA) ||     \
     defined(USE_ROCM)
 static void LogDeprecationWarning(
-    const std::string& deprecated, const optional<std::string>& alternative = nullopt) {
+    const std::string& deprecated, const optional<std::string>& alternative = std::nullopt) {
```

Comment:
it seems confusing to mix std::nullopt with onnxruntime::optional. until we can switch to std::optional completely, should we just keep using onnxruntime::optional and onnxruntime::nullopt?